core/jazz_environment-event-handler/config/prod-config.json

```diff
@@ -54,5 +54,13 @@
     "website": "build-pack-website",
     "sls-app": "build-pack-sls-app",
     "ui": "jazz_ui"
+  },
+  "TVAULT": {
+    "IS_ENABLED": true,
+    "ENDPOINT": "{endpoint_url}",
+    "API": "{t-vault_api}",
+    "CREATE_SAFE_API_RESOURCE": "/jazz/t-vault/safes",
```

Comment:
Do we need all of these? Also, should these be hardcoded?

Reply:
Why these changes are not in other config files?

Reply:
HOSTNAME and Tvault check are from config-loader. APIs endpoint is hardcoded and copied in the same all files.